### ISSUE TITLE
panlint: wrong message when template declaration is not on first line

### ISSUE THREAD

**Issue by wdpypere** [quattor/pan#224]
we ran into this:
```
cfg/os/c7x-x86_64/config/core/boot.pan:5: Missing space before and after operator
unique template config/core/boot;
                     ^^^  ^^^
```
when there was documentation above the declaration like:
```
@{
 comments
}

unique template config/core/boot;
```

although it is fine panlint nags about this, it should probably say the declaration needs to be on the first line instead of parsing the line.

**Comment by wpoely86** [comment]
```
cfg/os/c7x-x86_64/config/core/boot.pan:4: First non-comment line must be the template type and name

cfg/os/c7x-x86_64/config/core/boot.pan:5: Missing space before and after operator
unique template config/core/boot;
```

The first error is clear no?

**Comment by wdpypere** [comment]
@wpoely86 I did not get that first error. (I think)

**Comment by wdpypere** [comment]
@wpoely86 maybe I did not get enough coffee. I'll close this and if it should pop up again I'll reopen.

**Comment by stdweird** [comment]
this is confusing, the second error is plain wrong. and there is a different wrt the annotations (i'm a bit surprised too)

**Comment by stdweird** [comment]
actually i'm not surprised. if you want to annotate the templet itself, teh annotation has to be put before the template declaration

**Comment by wpoely86** [comment]
So it should 'detect' the annotation and skip that part for the 'first line' check?

**Comment by stdweird** [comment]
yes, you should treat annotations like comments (actually, i'd rather prefer not to allow comments before the declaration :wink:)

**Comment by wpoely86** [comment]
A wait, it does that. The issue is the empty line between the annotation and the template declaration.

So if we just add a 'skip' empty lines on the top, it should be fine again.

**Comment by stdweird** [comment]
no, you can nag about the emtpy lines, that is fine for me (but ideally, th emessage should say so)

**Comment by jrha** [comment]
This should be fixed by #231.
